engine/execution/execution_test.go

```diff
@@ -566,6 +566,10 @@ func TestBroadcastToMultipleVerificationNodes(t *testing.T) {
 			receipt, _ = args[2].(*flow.ExecutionReceipt)
 
 			assert.Equal(t, block.ID(), receipt.ExecutionResult.BlockID)
+			for i, chunk := range receipt.ExecutionResult.Chunks {
+				assert.EqualValues(t, i, chunk.CollectionIndex)
+				assert.Greater(t, chunk.TotalComputationUsed, uint64(10))
```

Comment:
```suggestion
				assert.Greater(t, chunk.TotalComputationUsed, uint64(0))
```

maybe just that is already enough. 10 is a bit confusing since its not obvious where its coming from.

Reply:
You are right, it's kind of a magic number :sweat_smile: